Assignment01/step-by-step.md

```diff
@@ -81,7 +97,27 @@ Now you can test whether you can call the VehicleRegistrationService. You can do
 
 1. Check the logging in the terminal window. It should look like this:
 
-   ![FineCollectionService logging](img/logging-finecollectionservice.png)
+   ```console
+   ❯ dotnet run
+   info: Microsoft.Hosting.Lifetime[0]
+         Now listening on: http://localhost:6001
+   info: Microsoft.Hosting.Lifetime[0]
+         Application started. Press Ctrl+C to shut down.
+   info: Microsoft.Hosting.Lifetime[0]
+         Hosting environment: Development
+   info: Microsoft.Hosting.Lifetime[0]
+         Content root path: D:\dev\Dapr\dapr-workshop\dotnet\FineCollectionService
+   info: System.Net.Http.HttpClient.Default.LogicalHandler[100]
+         Start processing HTTP request GET http://localhost:6002/vehicleinfo/RT-318-K
+   info: System.Net.Http.HttpClient.Default.ClientHandler[100]
+         Sending HTTP request GET http://localhost:6002/vehicleinfo/RT-318-K
+   info: System.Net.Http.HttpClient.Default.ClientHandler[101]
+         Received HTTP response headers after 148.5761ms - 200
+   info: System.Net.Http.HttpClient.Default.LogicalHandler[101]
+         End processing HTTP request after 156.5675ms - 200
+   info: FineCollectionService.Controllers.CollectionController[0]
+         Sent speeding ticket to Cassi Dakes. Road: A12, Licensenumber: RT-318-K, Vehicle: Mercedes SLK, Violation: 15 Km/h, Fine: 130 Euro, On: 20-09-2020 at 08:33:41.
```

Comment:
Nice! This line was cropped out of the screenshot, now it's here in its entirety.